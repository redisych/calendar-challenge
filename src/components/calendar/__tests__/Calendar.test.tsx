import * as React from 'react';
import * as enzyme from 'enzyme';
import { Calendar } from '..';
import toJson from 'enzyme-to-json';

const getWrapper = (): any => {
    return enzyme.shallow(
        <Calendar />
    );
};

describe('Calendar', () => {
    it('should be defined', () => {
        const wrapper = getWrapper();

        expect(wrapper).toBeDefined();
    });

    it('should match snapshot', () => {
        const wrapper = getWrapper();

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
