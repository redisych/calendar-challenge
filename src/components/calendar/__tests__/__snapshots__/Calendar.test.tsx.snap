// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar should match snapshot 1`] = `
<div
  className="calendar"
>
  <TimeScale />
  <EventsLayout />
</div>
`;
